sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/LogsHelper.cs

```diff
@@ -111,19 +129,25 @@ internal static (List<TelemetryItem> TelemetryItems, TelemetrySchemaTypeCounter
             return (telemetryItems, telemetrySchemaTypeCounter);
         }
 
-        internal static void ProcessLogRecordProperties(LogRecord logRecord, IDictionary<string, string> properties, out string? message, out string? eventName, out string? microsoftClientIp)
+        internal static void ProcessLogRecordProperties(LogRecord logRecord, IDictionary<string, string> properties, out string? message, out string? eventName, out string? microsoftClientIp, out AvailabilityInfo? availabilityInfo)
         {
             eventName = null;
+            availabilityInfo = null;
             message = logRecord.Exception?.Message ?? logRecord.FormattedMessage;
             microsoftClientIp = null;
+            bool hasAvailabilityData = false;
 
             foreach (KeyValuePair<string, object?> item in logRecord.Attributes ?? Enumerable.Empty<KeyValuePair<string, object?>>())
             {
                 if (item.Key == CustomEventAttributeName)
                 {
                     eventName = item.Value?.ToString();
                 }
-
+                else if (item.Key == AvailabilityNameAttributeName)
+                {
+                    hasAvailabilityData = true;
+                    break;
```

Comment:
Is other properties ignored if `AvailabilityNameAttributeName` is present?

Reply:
I mean, because it breaks out of loop at this line, all remaining items (they could have keys like `CustomEventAttributeName` and `ClientIpAttributeName`) will be ignored if the first item happens to be `AvailabilityNameAttributeName`.

Reply:
There are currently no plans to support both `AvailabilityNameAttributeName` and `CustomEventAttributeName` simultaneously. This attribute is set by the shim, and custom logs typically do not include any attributes with this name.